services/galley/test/integration/API/MLS.hs

```diff
@@ -1701,7 +1700,7 @@ testSelfConversationList isBelowV3 = do
         responseJsonError
           =<< listEndpoint u Nothing (Just 100)
             <!! const 200 === statusCode
-      pure $ foldr (<|>) Nothing $ guard . isMLSSelf u <$> mtpResults convIds
+      pure $ foldr ((<|>) . guard . isMLSSelf u) Nothing (mtpResults convIds)
```

Comment:
I find the previous version easier to read.

Reply:
This is an hlint rule to apply functor laws iirc